### PR TITLE
Update index.html and Added my Card

### PR DESCRIPTION
**Pull request by Sanyam-2026**
This is My First Open Source Contribution

### CODE REVIEW

**Review by Preet-Sojitra — APPROVED** [review]
LGTM 👍 
Your PR will be merged soon 
Thank you.

File: index.html [hunk 1]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+        <!-- ________ Sanyam Jain's card START ________  -->
+        <div class="card">
+          <p class="name">Sanyam Jain</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">@IasSanyamJain</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
            <a href="https://www.twitter.com/IasSanyamJain" target="_blank">@IasSanyamJain</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
kindly do the following changes

**Comment by Sanyam-2026** [comment]
> kindly do the following changes

done

**Comment by TenTraicion** [comment]
@Sanyam-2026, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**